providers/azure/connection/storage.go

```diff
@@ -60,7 +60,10 @@ func (sa *AzureStorageAccount) getStorageAccountClient(creds AzureCredentials) (
 
 // Create starts creation of a new Storage Account and waits for the account to be created.
 func (sa *AzureStorageAccount) Create(accountName, accountGroupName string, tags map[string]*string, httpsOnly bool, networkRuleSet *storage.NetworkRuleSet) (storage.Account, error) {
-
+	resourceLocation, err := azure.ResourceLocation()
+	if err != nil {
+		return storage.Account{}, err
+	}
```

Comment:
Mostly the SDK changes here are applied in [this PR to the AKS pack](https://github.com/probr/probr-pack-aks/pull/3), but here is an example of how it behaves. Retrieve the config var in the same way as before, but now for required vars there is an err returned if it isn't provided.